docs/user-guide/src/security_policy.md

```diff
@@ -0,0 +1,127 @@
+timeline can be communicated to users. Emergency releases for critical and high
+severity issues or fixes for issues already made public may affect the below
+timelines for how quickly or far in advance notifications will occur.
+
+The SL will lead the process of creating a GitHub security advisory for the
+repository that is affected by the issue. In case the SL has no administrator
+privileges the advisory will be created in cooperation with a repository admin.
+SL will have to request a CVE number for the security advisory.
+As GitHub is a CVE Numbering authority (CNA) there is an option to either use an
+existing CVE number or request a new one from GitHub. More about the GitHub
+security advisory and the CVE numbering process can be found
+[here](https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/about-repository-security-advisories).
+
+The original reporter(s) of the security issue has to be notified about the
+release date of the fix and the advisory and about both the content of the fix
+and the advisory as soon as the SL has decided a date for the fix disclosure.
+
+If a repository that has a release process requires a high severity fix then the
+fix has to be released as a patch release for all supported release branches
+where the fix is relevant as soon as possible.
+
+In case the repository does not have a release process, but it needs a critical
+fix then the fix has to be merged to the main branch as soon as possible.
+
+In repositories that have a release process Medium and Low severity
+vulnerability fixes will be released as part of the next upcoming minor or major
+release whichever happens sooner. Simultaneously with the upcoming release the
+fix also has to be released to all supported release branches as a patch release
+if the fix is relevant for given release.
+
+In case the fix was developed on a private repository either the SL or someone
+designated by the SL has to cherry-pick the fix and push it to the public
+repository. The SL and the Fix Team has to be able to push the PR through the
+public repo's review process as soon as possible and merge it.
+
+## Metal3 security committee members
+
+| Name              | GitHub ID           | Affiliation                    |
+|-------------------|---------------------|--------------------------------|
+| Dmitry Tantsur    | dtantsur            | Red Hat                        |
+| Riccardo Pittau   | elfosardo           | Red Hat                        |
+| Furkat Gofurov    | furkatgofurov7      | Ericsson Software Technologies |
+| Kashif Khan       | kashifest           | Ericsson Software Technologies |
+| Lennart Jern      | lentzi90            | Ericsson Software Technologies |
+| Zane Bitter       | zaneb               | Red Hat                        |
+| Tuomo Tanskanen   | tuminoid            | Ericsson Software Technologies |
+| Adam Rozman       | Rozzii              | Ericsson Software Technologies |
+
```

Comment:
It would be better to list Security committee members according to their org: Redhat and Ericsson

Reply:
I have changed it, is it now okay?

Reply:
Thanks. Looks good